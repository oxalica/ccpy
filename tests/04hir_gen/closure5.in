def f():
  global x
  nonlocal x
