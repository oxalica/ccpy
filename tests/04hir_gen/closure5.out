Gen fail: Declare global name as nonlocal
